cmd/cli/kubectl-kyverno/commands/test/test.go

```diff
@@ -281,6 +329,168 @@ func runTest(out io.Writer, testCase test.TestCase, registryAccess bool) (*TestR
 	return &testResponse, nil
 }
 
+func applyImageVerificationPolicies(
+	ivps []policiesv1alpha1.ImageValidatingPolicy,
+	jsonPayloads []*unstructured.Unstructured,
+	resources []*unstructured.Unstructured,
+	namespaceProvider func(string) *corev1.Namespace,
+	userInfo *kyvernov2.RequestInfo,
+	rc *processor.ResultCounts,
+	dclient dclient.Interface,
+	registryAccess bool,
+	contextPath string,
+	continueOnFail bool,
+) ([]engineapi.EngineResponse, error) {
+	provider, err := celengine.NewIVPOLProvider(ivps)
+	if err != nil {
+		return nil, err
+	}
+	var lister k8scorev1.SecretInterface
+	if dclient != nil {
+		lister = dclient.GetKubeClient().CoreV1().Secrets("")
+	}
+	engine := celengine.NewImageVerifyEngine(
```

Comment:
Should this function/struct naming also be updated?

Reply:
Fixed